content/graphing/miscellaneous/functions.md

```diff
@@ -223,6 +223,10 @@ Delta value between points for a given metric
 
 ## Smoothing
 
+### asap()
+
+The `asap()` function applies [ASAP algorithm][5] with the same window size for all the metrics in a graph. 
```

Comment:
I think we should indicate what it is that ASAP does (even if we don't describe how it does it). For example,

> The `asap()` function applies a moving average to the time series, using the [ASAP algorithm][5] to choose the optimal span for the moving average. When applied to a group-by query,`asap()` will use the same span for all groups.

I recommend that we use "span" instead of "window size" for consistency with the documentation for ewma below.